checkov/terraform/runner.py

```diff
@@ -115,8 +124,22 @@ def run_block(self, entities, definition_context, full_file_path, report, scanne
                 entity_lines_range = [entity_context.get('start_line'), entity_context.get('end_line')]
                 entity_code_lines = entity_context.get('code_lines')
                 skipped_checks = entity_context.get('skipped_checks')
-                variables_evaluations = definition_context[full_file_path].get('evaluations')
-                if variables_evaluations:
+
+                if full_file_path in self.evaluations_context:
+                    # {
+                    #     'region': {
+                    #         'var_file': '/tf/example.tf',
+                    #         'value': 'us-east-1',
+                    #         'definitions': [{
+                    #             'definition_name': 'region',
+                    #             'definition_expression': '${var.region}',
+                    #             'definition_path': 'resource/0/aws_s3_bucket/foo-bucket/region/0'
+                    #         }]
+                    #     }
+                    # }
```

Comment:
@robeden What's this? Is it `self.evaluations_context[full_file_path]`?

Reply:
Yes, sorry... that's some old guidance I left for myself. That structure really took me a while to wrap my head around, so I left it in there to remind me. I'm good to pull it now.